packages/code-du-travail-api/src/server/routes/__tests__/search.test.js

```diff
@@ -15,7 +15,10 @@ const app = new Koa();
 app.use(router.routes());
 
 it("asks same sources wether it is search sem or search elastic and gets a description", () => {
```

Comment:
@lionelB This should do the trick ?

Reply:
nice trick !  We will probably need to update mock data but for now, job is done !